Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/BoundingBox/BoundingBox.cs

```diff
@@ -936,12 +1065,14 @@ public GameObject Target
             }
         }
 
-        public BoxCollider TargetBounds
-        {
-            get { return cachedTargetCollider; }
-        }
+        /// <summary>
+        /// The collider reference tracking the bounds utilized by this component during runtime
+        /// </summary>
+        public BoxCollider TargetBounds { get; private set; }
+
+        private List<Handle> handles;
 
-        private List<Vector3> inputPoints = new List<Vector3>();
+        private List<Transform> corners;
```

Comment:
Should this be initialized?

Reply:
If so same for balls and links. There’s a deep rabbit hole in this file to fix/change a lot